server/package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "near-transactions-accounting-report",
   "version": "0.1.0",
-  "main": "src/index.js",
+  "main": "src/index",
   "type": "module",
   "scripts": {
-    "start": "node src/index.js",
-    "start-dev": "nodemon -r dotenv/config src/index.js dotenv_config_path=.env.development.local",
+    "start": "node src/index",
+    "dev": "node -r dotenv/config --loader ts-node/esm src/index.ts dotenv_config_path=.env.development.local",
```

Comment:
Does `--loader ts-node/esm` replaces `nodemon`?

Reply:
@sandoche  Whatever the original setup was was causing my browser to open a new tab upon every file save (instead of hot-reloading the tab). 

I can't remember whether hot-reloading works yet, but I definitely wanted to stop whatever was opening new tabs because it was super annoying.

Reply:
Ok got it!